dts/bindings/timer/st,stm32-timers.yaml

```diff
@@ -15,7 +15,6 @@ properties:
       required: true
 
     interrupts:
-      required: false
```

Comment:
Need to check for empty entries, but more importantly, it **can** be that you override a base `required: true` property which makes it not redundant.

Reply:
Hmm... it might potentially be better to add an option to warn for redundant `required: falsee` into `dtc`, unless we have access to the full AST from python.

Reply:
Hmm.. I wonder if it is a good idea to actually override a default required true... It may have some usecases though.

Reply:
Wait, what do you mean? You can't "un-require" a property.